### PR TITLE
[Docs] Control the display of the intra-page navigator toolbar

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

**Notes for Reviewers**

This PR enhances the Meshery Docs and it's ability to control the display of the intra-page navigator or not.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@sandramsc your review is encouraged.

**Comment by l5io** [comment]
🚀 Preview for commit 56d9e7835e01b88854cc562cbd5ae1e34157fc5e at: https://6514aab0fc4caa12ec9140ef--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-docs.md [hunk 1]
```diff
@@ -296,6 +296,14 @@ The output of the code snippet would be:
...
 
+### Editable Intra-page Table of Contents Toolbar
+
+Control the display of this intra-page navigator with either page level or layout level frontmatter variables:
+
+`display-toolbar`
+
+Set to `true` (make "editable" toolbar visible) or `false` (hide "editable" toolbar)
+
 ### `if` condititional
```
Comment (sandramsc) [inline docs/pages/project/contributing/contributing-docs.md:307]: Perhaps not a merge blocker, however I suggest adjusting this word to fix this minor grammar error
```suggestion
### `if` conditional
```

**Comment by sandramsc** [comment]
@leecalcote , the 
**MeshMap Screenshot Service / MeshMapScreenshot (pull_request_target)**  check is failing perhaps because the format of the environment variable "b0db3617-d44a-4d04-b66f-1af31a9593e2" is not set correctly? 
p1

**Comment by l5io** [comment]
🚀 Preview for commit 8f98a53e28ac07046497e9bf945cd8931f1ea6d9 at: https://65160eda13a91b0a17d463f3--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
> @leecalcote , the
> **MeshMap Screenshot Service / MeshMapScreenshot (pull_request_target)** check is failing perhaps because the format of the environment variable "b0db3617-d44a-4d04-b66f-1af31a9593e2" is not set correctly?
> imageedit_5_3961480814

Thank you much for pointing this out, @sandramsc. It certainly seems that way. I'll have a look...

File: docs/pages/project/contributing/contributing-docs.md (hunk 1 above)
Comment (leecalcote) [reply docs/pages/project/contributing/contributing-docs.md:307]: Oh, thank you. Very good.

**Comment by l5io** [comment]
🚀 Preview for commit adc00adf37574cae67b916853efc166846cbadc3 at: https://6518e0b358a22706f5a471fa--meshery-docs-preview.netlify.app

**Pull request merged**